app/controllers/trainees/training_routes_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Trainees
+  class TrainingRoutesController < ApplicationController
+    before_action :ensure_trainee_is_draft!
+
+    def edit
+      authorize trainee
+    end
+
+    def update
+      authorize trainee
+      trainee.update!(trainee_params)
+      redirect_to trainee
+    end
+
+  private
+
+    def trainee_params
+      params.fetch(:trainee, {}).permit(:training_route)
```

Comment:
You should always have a trainee param:

```suggestion
      params.require(:trainee).permit(:training_route)
```